docs/installationguide.md

```diff
@@ -165,16 +165,29 @@ JavaScript file, that contains the following sections:
 The `config.mqtt` section of the config file contains all the information needed to connect to the MQTT Broker from the
 IoT Agent. The following attributes are accepted:
 
+-   **protocol**: protocol to use for connecting with the MQTT broker (`mqtt`, `mqtts`, `tcp`, `tls`, `ws`, `wss`).
```

Comment:
It would be usefault to explain which one is the default (I understand that `mqtt`, for backward compatibility).

Reply:
Fixed a4adf6e